opencog/rule-engine/Rule.cc

```diff
@@ -484,6 +484,30 @@ std::string Rule::to_string(const std::string& indent) const
 	return ss.str();
 }
 
+bool Rule::has_name_capture() const
+{
+    HandleSeq vardecls = VariableListCast((this->get_vardecl()))
+            ->get_variables().varseq;
```

Comment:
The indendataion beig used here seems incosistent with the rest of the file (the rest of the file uses tabs.)